### DISCUSSION TITLE
Community Wiki, independent of GitHub

### DISCUSSION THREAD

**Discussion by Alxira5** [SuperTux/supertux#2691] [Ideas]
Hello community, I start this discussion because I think it would be better to develop a wiki for the community and independently of GitHub, since a GitHub wiki can be a bit annoying for players as opposed to developers.

When viewing the wiki of the moment, combines a wiki for developers and a wiki for players, and I think it should be a little more practical and with an easier to use interface.

In my opinion, we can use a framework like [Docusaurus](https://docusaurus.io/) and [Jekyll](https://jekyllrb.com/), or another framework that is useful and fulfills the function.

Greetings to the whole community.

**Comment by tylerandari13** [comment]
I dont hate this idea, but my only question is if custom content (such as custom enemies and thkngs related to addons) would be allowed. If players can edit the wiki then its highly likely theyre gonna add their addons and enemy concepts, and jugding by what people tell me about the forums this custom content would probably be... less than stellar.

**Comment by provigz** [comment]
IMO the GitHub wiki is way easier to maintain in the sense of automation, for example for auto-generating the scripting reference, since it is essentially a Git repository in itself.

The player and developer documentation issue isn't actually an issue IMO, since the wiki sidebar organizes the main wiki categories in various sections. Also having one wiki for everything keeps things organized in one place.
